src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -0,0 +1,107 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import frc.robot.config.Config;
+
+public class ShooterSubsystem extends SubsystemBase {
+
+  // Singleton instance for ShooterSubsystem
+  private static final ShooterSubsystem INSTANCE_SHOOTER = new ShooterSubsystem();
```

Comment:
Instead of initializing here, consider using a Singleton pattern that lazy loads instead of relying on static loading like this: https://github.com/georgetzavelas/guava-examples/blob/master/src/com/design/pattern/creational/singleton/Singleton.java